.kokoro/common.cfg

```diff
@@ -11,7 +11,7 @@ action {
 gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
 
 # Use the trampoline script to run in docker.
-build_file: "repo-automation-bots/.kokoro/trampoline.sh"
+build_file: "repo-automation-bots/.kokoro/trampoline_v2.sh"
```

Comment:
Right now, `trampoline.sh` is running the `populate-secrets.sh` then calling `trampoline_v2.sh`

Reply:
Oh, I see, `.trampolinerc` is calling `populate-secrets.sh`